website/docs/about/Governance.md

```diff
@@ -7,43 +7,113 @@ import Disclaimer from '../\_disclaimer.mdx';
+* If there are objections then they are discussed and, hopefully, resolved with full consensus. In the rare case of objections that cannot be resolved by consensus then meeting chair will hold a vote. A simple majority is required to accept the change. 
+* All meetings are recorded, transcribed, summarised, and published to the [UNTP meetings](https://uncefact.github.io/spec-untp/docs/about/Meetings) page.
+* A google group mailing list is also maintained and can be used by any observer or contributing member. All group emails are archived and searchable. 
+
+For participants less familiar with GitHub tools and processes, there is a [guidance page](https://github.com/uncefact/spec-untp#spec-untp) on how to write content and how to request changes via the pull request workflow. 
+
+## UNTP Version and Release Management
+
+Within the UNTP business governance framework, there also needs to be some technical governance to ensure quality and stability of UNTP technical deliverables.
+
+### Version Management
+
+All UNTP artifacts are rigorously versioned following [semver](https://semver.org/) best practices. 
+
+* Version numbers are indicated as a dot-separated triple `{major}.{minor}.{patch}`.  For example version 2.3.4.
+* `{patch}` version number increments indicate non-breaking bug fixes that do not add new capabilities of features. For example, implementers should see no difference between version 1.4.5 and version 1.4.6.
+* `{minor}` version number increments indicate non-breaking enhancements. For example, implementations of version 1.4.5 are still compatible with version 1.5.0 but may not take advantage of new features.
+* `{major}` version number increments indicate significant and breaking releases. For example implementations of version 1.5.0 will be incompatible with version 2.0.0 and may fail in unpredictable ways.
+
+Note that 0.x.y versions do not strictly follow semver and may include breaking changes in minor versions. However all versions after 1.0.0 first formal release will strictly observe this versioning process. 
+
+### Release Management
+
+Every version change is automatically published to the UNTP test end point following a defined URL structure
+
+**Linked data vocabulary (test)**
+
+* Pattern:  `https://test.uncefact.org/vocabulary/untp/{vocab-name/{major-version}/{artefact}`
+* Example: `https://test.uncefact.org/vocabulary/untp/dpp/0/Product`
+
+**Schema and context files (test)**
+
+* Pattern: `https://test.uncefact.org/vocabulary/untp/{credential-type}/{versioned-file-name}`
+* Example: `https://test.uncefact.org/vocabulary/untp/dpp/untp-dpp-schema-0.5.0.json`
+
+When a given version meets criteria to justify a production release then the governance process will approve a release. Not all versions will be released to production. Production releases will be published as follows.
```

Comment:
So the examples below are specifically of a major version release... is that what is meant by a "production" release here, or do you mean any release that will update production assets (such as, I assume, a minor release). It's not clear to me whether the "not all versions will be released to production" is meaning that some minor releases won't update these URLs, or that minor releases will result in changes to the content of these URLs but no new URLs are published?